app/components/my-map/show-all-observations/component.js

```diff
@@ -35,16 +35,16 @@ export default class MyMapShowAllObservationsComponent extends Component {
   @action
   markerTooltipOpen(observation) {
     if (this.isOpenObservationId) {
-      if (this.isOpenObservationId !== observation.get('id')) {
-        this.isOpenObservationId = observation.get('id');
+      if (this.isOpenObservationId !== observation.id) {
```

Comment:
You are using just `id` from `observation`. Consider to destructure this prop at the top